java/ql/lib/ext/java.util.concurrent.model.yml

```diff
@@ -26,9 +32,15 @@ extensions:
       pack: codeql/java-all
       extensible: neutralModel
     data:
+      - ["java.util.concurrent", "CompletableFuture", "completeExceptionally", "(Throwable)", "manual"]
+      - ["java.util.concurrent", "CompletableFuture", "isDone", "()", "manual"]
+      - ["java.util.concurrent", "CountDownLatch", "await", "", "manual"]
       - ["java.util.concurrent", "CountDownLatch", "countDown", "()", "manual"]
+      - ["java.util.concurrent", "Executor", "execute", "(Runnable)", "manual"]
```

Comment:
This _could_ be a command injection sink, but we could catch that at the `Runnable` creation too I guess.

Reply:
Just to clarify, should this still be a neutral summary?

Reply:
https://github.com/github/codeql/pull/11968#issuecomment-1469666788